### ISSUE TITLE
CORS configuration for rest server missing

### ISSUE THREAD

**Issue by webmaster128** [CosmWasm/wasmd#222]
Up to wasmd 0.9.x we used the CORS flag

```
docker exec "$CONTAINER_NAME" \
  wasmcli rest-server \
  --node tcp://localhost:26657 \
  --trust-node \
  --cors \
  --laddr "tcp://0.0.0.0:$REST_PORT_GUEST" \
  > "$REST_SERVER_LOGFILE" &
```

to allow requests from any URL in browsers. With `0.10.0-alpha` , the `--cors` flag was removed (_ERROR: unknown flag: --cors_).

There is now an `--unsafe-cors` [flag documented in Cosmos SDK](https://docs.cosmos.network/master/interfaces/rest.html#cross-origin-resource-sharing-cors). But when I try to use it with wasmd 0.10.0-alpha right after --trust-node, I get the error _ERROR: unknown flag: --unsafe-cors_.

Is there a way to configure CORS to allow all origins?

**Comment by ethanfrey** [comment]
This was added in master, but not in launchpad: https://github.com/cosmos/cosmos-sdk/pull/5671

I will discuss to see if we can get this in launchpad 0.39.1

**Comment by ethanfrey** [comment]
It will be in launchpad v0.39.1: https://github.com/cosmos/cosmos-sdk/pull/6853

`--unsafe-cors` is exposed in https://github.com/CosmWasm/wasmd/pull/224 (please let me know if this works for you and I will merge to master)

**Comment by ethanfrey** [comment]
Closed by #224
